lib/gosquared/index.js

```diff
@@ -90,21 +91,62 @@ GoSquared.prototype.page = function(page){
 
 GoSquared.prototype.identify = function(identify){
   var traits = identify.traits({
-    userId: 'user_id',
-    created: 'created_at'
+    createdAt: 'created_at',
+    firstName: 'first_name',
+    lastName: 'last_name',
+    title: 'company_position',
+    industry: 'company_industry'
+  });
+
+  // https://beta.gosquared.com/docs/tracking/api/#properties
+  var special = [
+    'id',
+    'email',
+    'name',
+    'first_name',
+    'last_name',
+    'username',
+    'description',
+    'avatar',
+    'phone',
+    'created_at',
+    'company_name',
+    'company_size',
+    'company_position',
+    'company_industry'
+  ];
+
+  // Segment allows traits to all be in a flat object
+  // GoSquared requires all custom properties to be in a `custom` object,
+  // separate from special properties
+  // this isn't ideal, but otherwise all Segment users would need to use `custom`
+  var props = {};
+  var customObj = is.object(traits.custom);
+  if (customObj) props.custom = traits.custom;
+
+  each(traits, function(name, value){
+    // don't double move `custom` (but allows non-object custom)
+    if (name === 'custom' && customObj) return;
+
+    if (special.indexOf(name) !== -1) {
+      props[name] = traits[name];
+    } else {
+      if (!props.custom) props.custom = {};
+      if (!(name in props.custom)) props.custom[name] = traits[name];
+    }
   });
```

Comment:
Condensing them down would probably help a bit with readability, but how would you do the automatic `custom` extending if it's an object?

Happy for this to be changed if you'd rather but I don't seem to be able to get it to work (and adding 4 new component dependences seems quite extreme!)